applicationset/services/plugin/plugin_service_test.go

```diff
@@ -31,19 +32,13 @@ func TestPlugin(t *testing.T) {
 	defer ts.Close()
 
 	client, err := NewPluginService(context.Background(), "plugin-test", ts.URL, token, 0)
-	if err != nil {
-		t.Errorf("unexpected error: %v", err)
-	}
+	require.NoError(t, err, "unexpected error")
 
 	data, err := client.List(context.Background(), nil)
-	if err != nil {
-		t.Errorf("unexpected error: %v", err)
-	}
+	require.NoError(t, err, "unexpected error")
```

Comment:
We probably don't need that extra "unexpected error" param, as default message might be sufficient.